automobile/nature.md

```diff
@@ -50,6 +50,16 @@ randomly selected in the list.
 - `enableBoundingObject`: Defines if the tree should have a bounding object (the
 bounding object is made of a simple cylinder).
 
+### FastTree
```

Comment:
`FastTree` is a strange PROTO name, unlike any other PROTO we have, as it refers to the speed of loading the PROTO and not the resulting object it actually is... I'd rather choose a more neutral name and explain in the documentation that it loads faster than the `SimpleTree` proto. Something like `CustomTree` or maybe, even better, simply `Tree` look better names to me.